communication/website/website.py

```diff
@@ -0,0 +1,73 @@
+from flask import Flask, render_template, redirect, url_for, request
+import management.boxes as box
+import random
+
+
+app = Flask(__name__)

```

Comment:
Please use a Blueprint for it. basically replace app with an instance of `BluePrint`.. you can register a blueprint in any app with a different prefix to compose apps accross multiple files